packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/zoe-upgrade/zoe-upgrade-driver.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+. ./upgrade-test-scripts/env_setup.sh
+
+set -euo pipefail
+
+here='upgrade-test-scripts/agoric-upgrade-11/zoe-upgrade'
+
+agd --chain-id=agoriclocal \
+  tx gov submit-proposal swingset-core-eval \
+  ${here}/zoe-upgrade-permit.json ${here}/zoe-upgrade-script.js \
+  --title="Zoe Upgrade" --description="zoe upgrade test" \
+  --deposit=10000000ubld \
+  --gas=auto --gas-adjustment=1.2 \
+  --yes -o json --from=validator --keyring-backend=test -b sync
```

Comment:
Looking at other `tx gov submit` stuff in this framework, I think a few of these flags are over-kill in this context. They were needed for devnet or something.